news/2024/2024-07-28-the-perennial-concludes.md

```diff
@@ -0,0 +1,420 @@
+
+Even more stunning was the first bout of grand finals; together we are terrific managed to force the bracket reset over the yet-undefeated APAC RADIANT with a 7-5 scoreline. For some on together we are terrific, this win was a long time coming; their captain, tekkito, has been third and second place in the Perennial 2021 and 2022, so victory here would finally mean the win after four years of effort. Unfortunately for together we are terrific, the members of APAC RADIANT wanted the win just as much, having never made the podium before 2024. APAC RADIANT took an early break point in the bracket reset and kept up the pressure, ultimately snatching victory with a 7-4 scoreline in the bracket rest.
+
+## Match highlights
+
+Each round, from group stage to grand finals, we uploaded highlight videos showcasing the best scores of the weekend, the closest matches, and the hypest maps. In addition, we have pulled the closest matches from throughout the tournament and linked all their vods below. Whether you just want the highlights, the abridged version, or the full experience, you can relive the Perennial 2024 on our [YouTube](https://youtube.com/c/ThePerennialYT) where all match vods are uploaded in full.
+
+### Match VODs
+
+| Match | Stage | Link |
+| :-- | :-- | :-- |
+| assalamualaikum vs. WELCOME TO TREV CITY | Group Stage | [VOD](https://youtu.be/Pfra41-02_s?si=YNkgiPBf_DZ3wB7Z) |
+| parmesan vs. the freak squad | Group Stage | [VOD](https://youtu.be/Tyhb1SI112o?si=k0vtNcFBUJVBnuAg) |
+| Tea Party vs. european goats | Round of 16 | [VOD](https://youtu.be/l9RhJOnIwJ8?si=0ztZkiWaArwjQv7P) |
+| mom's touch vs. Nipis Madu | Round of 16 | [VOD](https://youtu.be/fFqyEZkjiO4?si=6HuwQc36xJ-SE7rA) |
+| Hatsune Miku vs. EMPICONES | Round of 16 | [VOD](https://youtu.be/Vk5slVJ6CiI?si=Kl5H4y0ofF3TT7GG) |
+| Nipis Madu vs. AHHHHHHHHHHHH | Quarter Finals | [VOD](https://youtu.be/xpR6bljg-i8?si=QR0n2L4C-zE3Eb8h) |
+| Bob vs. european goats | Quarter Finals | [VOD](https://youtu.be/tL1zR-bQqrM?si=FjzRmtxdNRKQwsZe) |
+| the freak squad vs. Hatsune Miku | Quarter Finals | [VOD](https://youtu.be/nQT0GVh9POk?si=g-QIV-WctbhtN0nd) |
+| widepeepoHappy vs. Tea Party | Semi Finals | [VOD](https://youtu.be/zHCkywcpUBs?si=o22FJLSbaf7SMdV9) |
+| together we are terrific vs. Hatsune Miku | Semi Finals | [VOD](https://youtu.be/3JwWWOKOhc8?si=8aSoB71_BGuCDNeN) |
+| mom's touch vs. EMPICONES | Finals | [VOD](https://youtu.be/5HcQRVBM8Iw?si=RGFIwtpbefvu1RU_) |
+| APAC RADIANT vs. together we are terrific | Grand Finals | [VOD](https://youtu.be/XRG7ewQExew?si=axH3ios8RxmJf62l) |
+
+### Highlight videos
```

Comment:
these are very tall to have all in a row. please embed a youtube playlist instead

example url: `https://www.youtube-nocookie.com/embed/videoseries?list=PLA2Nwtxa4WMqNujKzed748MxnJ0BfZ6YY`